app/views/layouts/_navbar.html.erb

```diff
@@ -1,20 +1,24 @@
-<div class="masthead clearfix masthead-fixed-top">
-  <div class="inner">
-    <h3 class="masthead-brand"><a href="/">Shotnr</a></h3>
-    <nav class="nav nav-masthead">
-      <%= link_to 'Shotlinks', links_path, class: 'nav-link' %>
-      <% if logged_in? %>
-        <%= link_to logout_path, class: 'nav-link' do %>
-          <span><i class="fa fa-sign-out" aria-hidden-"true"></i> Logout</span>
-        <% end %>
-        <%= link_to "#", class: 'nav-image-link' do %>
-          <%= image_tag current_user.image_url, class: 'profile-picture' %>
-        <% end %>
-      <% else %>
-        <%= link_to '/auth/twitter', class: 'nav-link' do %>
-          <span class="btn btn-info-outline btn-sm"><i class="fa fa-twitter"></i> Log In</span>
-        <% end %>
-      <% end %>
-    </nav>
+<div class="row">
```

Comment:
**UI-Critical**
Make the UI responsive for mobile devices. Shift to use bootstrap grid system to make the application responsive.